try:
    a = int(input("Enter the Number : "))
    print(a + 4)
except Exception as e:
    print("Some Errors Occurs " , e)